src/ext/java/org/opentripplanner/ext/legacygraphqlapi/generated/graphql-codegen.yml

```diff
@@ -30,7 +30,7 @@ config:
     AlertEffectType: String
     AlertSeverityLevelType: String
     BikePark: org.opentripplanner.routing.bike_park.BikePark#BikePark
-    BikeRentalStation: org.opentripplanner.routing.vehicle_rental.VehicleRentalStation#VehicleRentalStation
+    BikeRentalStation: org.opentripplanner.routing.vehicle_rental.VehicleRentalPlace#VehicleRentalPlace
```

Comment:
Is a `VehicleRentalPlace` what used to be called `VehicleRentalStation`?

Reply:
Yes, It has two classes implementing it, `VehicleRentalStation` and `VehicleRentalVehicle`